nix/wire-server.nix

```diff
@@ -79,7 +79,7 @@ let
     gundeck = [ "gundeck" "gundeck-integration" "gundeck-schema" ];
     proxy = [ "proxy" ];
     spar = [ "spar" "spar-integration" "spar-schema" "spar-migrate-data" ];
-    stern = [ "stern" ];
+    stern = [ "stern" "stern-integration"];
```

Comment:
the repo needs to be created upstream in quay.io: https://quay.io/repository/wire/stern-integration does not yet exist. let me create it ...